### ISSUE TITLE
Luminosity ratio of focus indicator on focused button in "More Actions..." in notification popup is 1.346:1 which is less than less than 3:1 in default dark theme.

### ISSUE THREAD

**Issue by miniwangdali** [microsoft/vscode#203400]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.85.2
- OS Version: Windows 11 Enterprise (Windows_NT x64 10.0.22631.3007)

```
Version: 1.85.2 (user setup)
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
Date: 2024-01-18T06:40:10.514Z
Electron: 25.9.7
ElectronBuildId: 26354273
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
```

**Description**
During an accessibility test, we found the contrast ratio between the highlight color of the focused button in "More Actions..." in notification popup and its background is not high enough under "Default Dark Modern" theme.

Highlight Color: `#043659`
Background Color: `#1D1D1D`
Contrast Ratio: 1.346:1
Required Contrast Ratio: 3:1

![image](https://github.com/microsoft/vscode/assets/5415452/738803f0-603a-4014-aecf-19b9d38b87e4)

**Comment by bpasero** [comment]
I think this refers to the background color of the gear icon in the toolbar of a notification relative to the background color of the notification.

**Comment by miniwangdali** [comment]
The bug reported on my side particularly specified the "Manage Extension" button highlight color and the background color of that actions popup under settings icon in the notification.

**Comment by shindepra** [comment]
#A11yMAS; #A11yTCS; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2023; #Non-text Contrast; #WCAG1.4.11

**Primary Bug ID** : https://microsoft.visualstudio.com/Xbox/_workitems/edit/48295664

**Comment by hbons** [comment]
Looks like it's about the blue on black bg, in that case this applies to all the web menus in VS Code.

**Comment by miniwangdali** [comment]
yes, it's about the blue highlight and black background.

**Comment by shindepra** [comment]
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2023; #WCAG1.4.11; #Non-text Contrast;

**Primary bug ID**:- https://microsoft.visualstudio.com/Xbox/_workitems/edit/48295664

**Comment by miniwangdali** [comment]
Hi, is there any update on the plan of fixing this accessibility bug?

**Comment by hbons** [comment]
putting this on the March plan.

**Comment by miniwangdali** [comment]
Hi, may I check if the fix is included in v1.88.0?

**Comment by miniwangdali** [comment]
@hbons Hi, I have checked v1.88.0 but it seems the active color is still `#04395E` not the one `#2F6DBB` you changed to in your PR. Did I miss something?

**Comment by rzhao271** [comment]
Verified with the Dark Modern theme, but do we also have to fix this with the Dark+ or Dark (Visual Studio) themes?
